articles/_a-propos/differences-utilisateur-organisatoion.md

```diff
@@ -3,6 +3,10 @@ title: Différences entre un utilisateur et une organisation
 slug: differences-utilisateur-organisation
 ---
 
+# 💡 Une nouvelle documentation est disponible ! 
+
+[Découvrez la nouvelle version beta des guides et de la documentation de data.gouv.fr](https://etalab-2.gitbook.io/guides/)
```

Comment:
We'll need to update this snippet as well as https://etalab-2.gitbook.io/guides/ when we're ready to migrate?